L'exécution de votre programme a déclenché une exception java.lang.RuntimeException : Désolé Dave, je ne peux pas te laisser utiliser setPos(x,y) dans cet exercice. Marche jusqu'à ton objectif.
 Merci de corriger votre code.
   at plm.runtime51.JediEscaper.setPos (plm.runtime51.JediEscaper:18)
   at plm.runtime51.JediEscaper.indicationCasesAdjacentes (plm.runtime51.JediEscaper:120)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1$$anonfun$apply$mcVI$sp$2.apply$mcVI$sp (plm.runtime51.JediEscaper:57)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1$$anonfun$apply$mcVI$sp$2.apply (plm.runtime51.JediEscaper:55)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1$$anonfun$apply$mcVI$sp$2.apply (plm.runtime51.JediEscaper:55)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply$mcVI$sp (plm.runtime51.JediEscaper:55)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply (plm.runtime51.JediEscaper:54)
   at plm.runtime51.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply (plm.runtime51.JediEscaper:54)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime51.JediEscaper$$anonfun$run$1.apply$mcVI$sp (plm.runtime51.JediEscaper:54)
   at plm.runtime51.JediEscaper$$anonfun$run$1.apply (plm.runtime51.JediEscaper:53)
   at plm.runtime51.JediEscaper$$anonfun$run$1.apply (plm.runtime51.JediEscaper:53)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime51.JediEscaper.run (plm.runtime51.JediEscaper:53)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (Thread.java:745)
